src/OmniSharp.MSBuild/ProjectLoader.cs

```diff
@@ -77,9 +77,27 @@ public ProjectLoader(MSBuildOptions options, string solutionDirectory, Immutable
 
                 var projectInstance = evaluatedProject.CreateProjectInstance();
                 var msbuildLogger = new MSBuildLogger(_logger);
+
+                var loggers = new List<MSB.Framework.ILogger>()
+                {
+                    msbuildLogger
+                };
+
+                if (_options.GenerateBinaryLogs)
+                {
+                    var binlogPath = Path.ChangeExtension(projectInstance.FullPath, ".binlog");
+                    var binaryLogger = new MSB.Logging.BinaryLogger()
+                    {
+                        CollectProjectImports = MSB.Logging.BinaryLogger.ProjectImportsCollectionMode.Embed,
```

Comment:
I think this is actually the default behaviour so maybe we can skip explicitly specifying it

Reply:
I figured the default behavior was "None": https://github.com/Microsoft/msbuild/blob/master/src/Build/Logging/BinaryLogger/BinaryLogger.cs#L47.

Reply:
Oh, I guess it is the default: https://github.com/Microsoft/msbuild/blob/master/src/Build/Logging/BinaryLogger/BinaryLogger.cs#L68

That's OK, let's keep the code specific so there's no question. I think our use case is definitely to Embed.